test/phabricator_test/test_phabricator.py

```diff
@@ -0,0 +1,356 @@
+from mock import patch
+import yaml
+import unittest
+from os.path import join, dirname
+try:
+    # Python 3.3 >
+    from unittest.mock import MagicMock
+except:
+    from mock import MagicMock
```

Comment:
@Lukkra211 we have a plan to stop supporting the python2 but we have to communicate with our customers and decide how we do it. But for now, it would be great to support both.

Reply:
Ok, it would be great if you could stop using v2.